dev/FHIR4-0-1-MedMij-Cert/Immunization-1-0/_reference/resources/imm-HealthcareProvider-05.xml

```diff
@@ -0,0 +1,128 @@
+        <valueCodeableConcept>
+          <coding>
+            <system value="urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.22.1"/>
+            <code value="LL"/>
+            <display value="Land Line"/>
+          </coding>
+        </valueCodeableConcept>
+      </extension>
+    </system>
+    <value value="+31612121214"/>
+    <use value="work"/>
+  </telecom>
+  <telecom>
+    <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-Comment">
+      <valueString value="Noodnummer"/>
+    </extension>
+    <system value="pager">
+      <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-CodeSpecification">
+        <valueCodeableConcept>
+          <coding>
+            <system value="http://terminology.hl7.org/CodeSystem/v3-AddressUse"/>
+            <code value="PG"/>
+            <display value="Pieper"/>
+          </coding>
+        </valueCodeableConcept>
+      </extension>
+    </system>
+    <value value="+31612121215"/>
+    <use value="work"/>
+  </telecom>
+  <telecom>
+    <system value="email"/>
+    <value value="hulplijn@testmailbarones.nl"/>
+    <use value="work"/>
+  </telecom>
+  <telecom>
+    <system value="email"/>
+    <value value="hulplijn2@testmailbarones.nl"/>
+    <use value="work"/>
+  </telecom>
+  <address>
+    <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-AddressInformation.AddressType">
+      <valueCodeableConcept>
+        <coding>
+          <system value="http://terminology.hl7.org/CodeSystem/v3-AddressUse"/>
+          <code value="WP"/>
+          <display value="Werkadres"/>
+        </coding>
+      </valueCodeableConcept>
+    </extension>
+    <use value="work"/>
+    <line value="Hoofdstraat to 50 A 02">
+      <extension url="http://hl7.org/fhir/StructureDefinition/iso21090-ADXP-streetName">
+        <valueString value="Hoofdstraat"/>
+      </extension>
+      <extension url="http://hl7.org/fhir/StructureDefinition/iso21090-ADXP-houseNumber">
+        <valueString value="50"/>
+      </extension>
+      <extension url="http://hl7.org/fhir/StructureDefinition/iso21090-ADXP-buildingNumberSuffix">
+        <valueString value="A 02"/>
+      </extension>
+      <extension url="http://hl7.org/fhir/StructureDefinition/iso21090-ADXP-unitID">
+        <valueString value="Zorghuis JF"/>
+      </extension>
+    </line>
+    <city value="Stitswerd"/>
+    <district value="Het Hogeland"/>
+    <postalCode value="9999PT"/>
+    <country value="Nederland">
+      <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-CodeSpecification">
+        <valueCodeableConcept>
+          <coding>
+            <system value="urn:iso:std:iso:3166"/>
+            <version value="2020-10-26T00:00:00"/>
+            <code value="NL"/>
+            <display value="Nederland"/>
+          </coding>
+        </valueCodeableConcept>
+      </extension>
+    </country>
+  </address>
+  <managingOrganization>
+    <reference value="Organization/imm-HealthcareProvider-Organization-05"/>
```

Comment:
Deze organisatie zie ik niet terug in de _reference van src

Reply:
Nu wel?